packages/react/src/components/dropdown-list/dropdown-list.tsx

```diff
@@ -131,13 +157,14 @@ export interface DropdownListProps {
     /**
      * Set the selected value
      */
-    value?: string;
+    value?: Value;
     hint?: string;
+    multiselect?: boolean;
```

Comment:
Ca serait pt mieux de meme, comme en html `<select multiple>`
```suggestion
    multiple?: boolean;
```

Reply:
Bonne suggestion, mais étant donné que c'est le nom qu'on utilise à l'interne depuis des années je le laisserais de même par souci de consistance.